Sources/NIOSSHServer/GlueHandler.swift

```diff
@@ -72,6 +72,12 @@ extension GlueHandler: ChannelDuplexHandler {
 
     func handlerAdded(context: ChannelHandlerContext) {
         self.context = context
+
+        // It's possible our partner asked if we were writable, before, and we couldn't answer.
+        // Consider updating it.
+        if context.channel.isWritable {
+            self.partner?.partnerBecameWritable()
```

Comment:
Is there a scenario where we became writable but the partner wasn't set yet? This also goes for `channelWritabilityChanged`.

Reply:
Seeing the current usages, and the fact that the class is internal, I think it's never going to happen. So it might be wise to make it implicitly unwrapped.

Reply:
In principle the partner should always be set, but we do nil it out explicitly on `handlerRemoved` so it’s likely better safe than sorry here.